tools/tpm2_flushcontext.c

```diff
@@ -253,13 +291,15 @@ static bool on_option(char key, char *value) {
         ctx.encountered_option_flags |= 1 << 0;
         break;
     case 'l':
-
         ctx.encountered_option_flags |= 1 << 1;
         ctx.encountered_option_flags++;
```

Comment:
@williamcroberts is this a typo?

Reply:
Yep, no more increment just set the flags